### PR TITLE
[DOCS] Remove outdated integrations and links from versioned docs

### PR DESCRIPTION
**Pull request by Rachel-Reverie**
## Description
- Removes outdated integrations from 0.17.x docs
- Removes outdated CLI references from 0.17.x docs
- Removes outdated links from 0.17.x docs
- Removes docs and component files that were orphaned by the above changes

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] Appropriate tests and docs have been updated

### CODE REVIEW

**Comment by ghost** [comment]
<details open>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=9355&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

####

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=9355&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
CodeSee Map legend
</details>

File: docs/docusaurus/versioned_docs/version-0.17.23/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md [hunk 1]
```diff
@@ -175,7 +173,7 @@ Using the  <TechnicalTag relative="../../../" tag="data_context" text="Data Cont
 
 In the example, we have created a Data Source named `my_bigquery_datasource`, using the `add_or_update_sql` method and passing in a connection string.
 
-To configure the BigQuery Data Source, see [How to connect to a BigQuery database](/docs/0.15.50/guides/connecting_to_your_data/database/bigquery).
+To configure the BigQuery Data Source, see [How to connect to a BigQuery database](/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md).
```
Comment (tyler-hoffman) [inline docs/docusaurus/versioned_docs/version-0.17.23/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md]: Starting the link with `/docs/` points it to the current version of docs. This should be changed to start with `/guides` I might have missed other instances of this in the review

**Review by tyler-hoffman — CHANGES_REQUESTED** [review]
I didn't verify that all the links/paths actually point to resources, but the shape of the changes looks right to me. Only blocking because of the one link that I caught that starts with `/docs/`

File: docs/docusaurus/versioned_docs/version-0.17.23/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md (hunk 1 above)
Comment (Rachel-Reverie) [reply docs/docusaurus/versioned_docs/version-0.17.23/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md]: I've corrected this in the links that this PR edits.

**Review by tyler-hoffman — APPROVED** [review]
Thanks!

**Pull request merged**